website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
Will removing
- /project/<pid>/retraction/approve/<token>/
- /project/<pid>/retraction/disapprove/<token>/
- /project/<pid>/embargo/approve/<token>/
- /project/<pid>/embargo/disapprove/<token>/

break existing url paths since they are already in production?